swim/handlers.go

```diff
@@ -121,9 +121,22 @@ func (n *Node) tickHandler(ctx json.Context, req *emptyArg) (*ping, error) {
 	return &ping{Checksum: n.memberlist.Checksum()}, nil
 }
 
-func (n *Node) adminJoinHandler(ctx json.Context, req *emptyArg) (*Status, error) {
-	n.memberlist.MakeAlive(n.address, util.TimeNowMS())
-	return &Status{Status: "rejoined"}, nil
+type Clock interface {
```

Comment:
i dont think it should be in `handlers.go` either. It's a generally useful abstraction.